specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/sqlPool.json

```diff
@@ -1564,6194 +1564,6194 @@
             "schema": {
               "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
             },
```

Comment:
This file is unreviewable due to the large amount of whitespace changes. Please revert the superfluous whitespace changes.